test/remote/account_signin_verification_enable_tests.js

```diff
@@ -9,8 +9,8 @@ var Client = require('../client')
 test(
   'signin confirmation can be disabled',
   function (t) {
-    process.env.SIGNIN_CONFIRMATION_ENABLED = false
     var config = require('../../config').getProperties()
+    process.env.SIGNIN_CONFIRMATION_ENABLED = false
```

Comment:
Did you mean to move this line? I imagine the side effect is for config to initially have signin confirmation enabled?